truss/remote/baseten/service.py

```diff
@@ -29,7 +29,12 @@ def is_live(self) -> bool:
     def is_ready(self) -> bool:
         raise NotImplementedError
 
-    def predict(self, model_request_body: Dict):
+    def predict(
+        self,
+        model_request_body: Dict,
+        model_id: Optional[str] = None,
+        model_version_id: Optional[str] = None,
+    ):
```

Comment:
Shouldn't we change the `_service_url` based on if you wanna invoke by model_id or model_version_id?

Reply:
I wasn't sure what the best way to handle this was. It felt weird to have to have two places where we process the `ModelIdentifier` (constructing the `BasetenService` AND in the `predict` function), so the way this works now is that the model_version_id gets resolved here in the baseten service, regardless of what you pass (model id, version id, or name), and we always invoke on that.

I can change predict to take a `ModelIdentifier` as well, but this felt kinda messy.

Reply:
why not just the `_service_url` based on the constructor of the BasetenService?

Reply:
cool -- fixed!